jdk/test/java/awt/font/TextLayout/TestSinhalaChar.java

```diff
@@ -21,7 +21,9 @@
  *
  */
 
-/* @test
+/**
+ * @test @(#)TestSinhalaChar.java
```

Comment:
"@(#)TestSinhalaChar.java" in this line seems unnecessary, it will revert the fix for https://bugs.openjdk.org/browse/JDK-8293767